marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/smart-mastering/merge-struct-prop/test-data/steps/merging/mergeCustomersStandard.step.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "mergeCustomersStandard",
+  "stepDefinitionName": "default-merging",
+  "stepDefinitionType": "merging",
+  "stepId": "mergeCustomersStandard-merging",
+  "targetEntityType": "http://example.org/Customer-0.0.1/Customer",
+  "description": "",
+  "selectedSource": "collection",
+  "sourceQuery": "cts.collectionQuery(['matchCustomers'])",
+  "collections": ["mergeCustomers"],
+  "additionalCollections": [ ],
+  "sourceDatabase": "data-hub-FINAL",
+  "targetDatabase": "data-hub-FINAL",
+  "targetFormat": "JSON",
+  "permissions": "data-hub-common,read,data-hub-common-writer,update",
+  "provenanceGranularityLevel": "fine",
+  "lastUpdatedLocation": {
+    "documentXPath": "/envelope/headers/createdOn"
+  },
+  "mergeStrategies": [
+  ],
+  "mergeRules": [
+  ],
+  "targetCollections": {
```

Comment:
Do you need these for the tests? I'm just thinking that if the main purpose is to verify that the merge strategies worked, then the collections are noise and should be omitted.

Reply:
Removed the targetCollections